backend/pubsub.js

```diff
@@ -301,7 +305,31 @@ const defaultMessageHandlers = {
     } else {
       log.debug('Already subscribed to', channelID)
     }
-    socket.send(createOkResponse({ type: SUB, channelID }))
+    // $FlowFixMe[incompatible-call]
+    socket.send(createOkResponse({ type: SUB, channelID, kvFilter }))
```

Comment:
`o3-mini-high` isn't happy with these `// $FlowFixMe[incompatible-call]` comments and asks if casts can be used, e.g. `(createOkResponse({ … }): string)`

Reply:
(serialising `undefined` as JSON is fine, it just omits the field, unlike `null`, which keeps the field with  a `null` value)